### ISSUE TITLE
use p.adj in stat_compare_means

### ISSUE THREAD

**Issue by jdkparker** [kassambara/ggpubr#11]
Hi I would like an option to use p.adj as the basis for p.format and display, is it possible?

**Comment by kassambara** [comment]
Hi,

I'm out of office and have limited access to my e-mail. I will be able to check this on jully 17th.

Thank you

**Comment by kassambara** [comment]
Of course, this is possible. Read this: [http://www.sthda.com/english/wiki/add-p-values-and-significance-levels-to-ggplots#compare-two-independent-groups](http://www.sthda.com/english/wiki/add-p-values-and-significance-levels-to-ggplots#compare-two-independent-groups)

Briefly, you can do this:

```r
p <- ggboxplot(ToothGrowth, x = "supp", y = "len",
          color = "supp", palette = "jco",
          add = "jitter")

p + stat_compare_means( aes(label = format.pval(..p.adj.., digits = 3)), 
                        label.x = 1.5, label.y = 40)
```

**Comment by kristineccles** [comment]
Hi,

I too am having trouble getting the p.adj to display on the boxplot. When I use the code you supplied, it still produces the unadjusted p-values. I've also noticed that p.signif is assigned based on the unadjusted p-values, not the adjusted p-values (as is the case in the screenshot I attached). This could lead people to believe there is a difference between groups, when there is not. Is there a way to display the adj p-value and/or assign significance codes based on the adj. p-value?

Thanks,
Kristin
<img width="623" alt="Screen Shot 2019-11-27 at 11 16 23 AM" src="https://user-images.githubusercontent.com/26681364/69740420-5cf19180-1107-11ea-83c0-835da1d9c065.png">
